apps/helpthechickens.com/src/app/page.tsx

```diff
@@ -61,21 +63,247 @@ const Hero = () => {
   );
 };
 
-const PetitionSection = () => {
+function PetitionSection() {
   return (
     <Section
       className="gap-12 items-center bg-slate-200 xl:rounded-lg py-12 md:px-16"
       id="petition-section"
     >
       <h2 className="font-semibold text-xl uppercase self-start text-center md:text-left w-full">
-        Ask Petaluma City Council to shut down Perdue's slaughterhouse
+        Ask Petaluma City Council to shut down Perdue&apos;s slaughterhouse
```

Comment:
i was wondering how eslint let you get away w/ this when i was reviewing earlier 😆

Reply:
yeah whoops. later but soon i wanna make the monorepo global linting configs and vs code linting settings.